### PR TITLE
Add stubs required by imageio

### PR DESCRIPTION
**Pull request by liranr23**
When importing with the populator it uses ovirt-imageio client. The client requires more stubs and some changes in the imagetransfer ones.

### CODE REVIEW

File: stubs/hosts/2900d577-63a8-4ec9-acff-8493278346b1/content.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+        }, {
+          "href" : "/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1/nics/78a019f2-262d-479d-93b8-cf17fce112a5/networkattachments",
+          "rel" : "networkattachments"
+        }, {
+          "href" : "/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1/nics/78a019f2-262d-479d-93b8-cf17fce112a5/statistics",
+          "rel" : "statistics"
+        } ]
+      } ]
+    },
+    "actions" : {
```
Comment (ahadas) [inline stubs/hosts/2900d577-63a8-4ec9-acff-8493278346b1/content.json]: I think you can drop the actions

File: stubs/hosts/2900d577-63a8-4ec9-acff-8493278346b1/content.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+      }, {
+        "href" : "/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1/refresh",
+        "rel" : "refresh"
+      } ]
+    },
+    "name" : "vhost",
+    "comment" : "",
+    "href" : "/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1",
+    "id" : "2900d577-63a8-4ec9-acff-8493278346b1",
+    "link" : [ {
```
Comment (ahadas) [inline stubs/hosts/2900d577-63a8-4ec9-acff-8493278346b1/content.json]: and probably the links as well (to have an empty list)

File: stubs/imagetransfers/stub.json [hunk 3]
```diff
@@ -4,8 +4,9 @@
     "method": "POST",
     "responses": [
       {
+	"responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><action><status>complete</status></action>",
         "responseCode": 200,
-        "times": 3
+        "times": 10000
```
Comment (ahadas) [inline stubs/imagetransfers/stub.json]: @bennyz mentioned a way to have it not limited at all - is it by dropping the "times" argument maybe?

File: stubs/storagedomains/95ef6fee-5773-46a2-9340-a636958a96b8/content.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+  "storage_domain": {
+    "actions": {
+      "link": [
+        "",
+        "",
+        "",
+        ""
```
Comment (ahadas) [inline stubs/storagedomains/95ef6fee-5773-46a2-9340-a636958a96b8/content.json]: ok, that's an indication that we can indeed drop these

File: stubs/imagetransfers/stub.json (hunk 3 above)
Comment (bennyz) [reply stubs/imagetransfers/stub.json]: looks like there's no support for it

https://github.com/kubev2v/fakeovirt/blob/30ea054013ed115bb9d57897216a6667f55ffe40/internal/app/handlers/dynamic/stubbing-handler.go#L12

Comment (liranr23) [reply stubs/imagetransfers/stub.json]: ok, i left it as it is

Comment (ahadas) [reply stubs/imagetransfers/stub.json]: ack, we'll change it separately

**Review by ahadas — APPROVED** [review]
@liranr23 nice!
about returning the specified value unlimited number of times - yeah, that can/should be done separately

**Pull request merged**